### PR TITLE
DOC Add section on security of supported libraries

### PR DESCRIPTION
**Pull request by BenjaminBossan**
This is to make it more transparent for users what they can expect when using skops with supported external libraries.

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
Ready for review @skops-dev/maintainers

**Review by adrinjalali — APPROVED** [review]
I'll let @E-Aho to also have a look here.

**Review by E-Aho — APPROVED** [review]
LGTM :)

**Pull request merged**
